internal/dataplane/parser/translate_httproute.go

```diff
@@ -167,11 +173,20 @@ func generateKongRoutesFromHTTPRouteRule(httproute *gatewayv1beta1.HTTPRoute, ru
 			if match.Path != nil {
 				switch *match.Path.Type {
 				case gatewayv1beta1.PathMatchExact:
-					terminated := *match.Path.Value + "$"
+					terminated := "^" + *match.Path.Value + "$"
```

Comment:
In kong 3.0, just `~match.Path.Value` is OK.  `~^/aaa$` is not accepted by Kong 3.0 router.